src/panels/Menu/TaskBar/TaskBarMenuButton.tsx

```diff
@@ -13,6 +21,10 @@ export function TaskBarMenuButton({ onClick, onRightClick, children, ...props }:
       size={'xl'}
       variant={'menubar'}
       {...props}
+      style={{
+        borderBottom: isOpen ? '4px solid var(--mantine-primary-color-filled)' : '',
```

Comment:
do we want to make a variable for this style?  Now it's used in three places :) (including the selected node in the scene tree)

Reply:
Is it possible to add something to the theme?

Reply:
Not that I'm aware of, at least

Reply:
And we use the border for different locations (left VS bottom)